packages/ui/src/shared-style-variables.css

```
@@ -668,6 +681,224 @@
+    --canary-accent: var(--canary-base-chrome-100);
+    --canary-accent-foreground: var(--canary-base-chrome-black);
+    --canary-destructive: var(--canary-indigo-600);
+    --canary-destructive-foreground: var(--canary-base-chrome-white);
+    --canary-border: var(--canary-base-chrome-150);
+    --canary-border-background: var(--canary-base-chrome-150);
+    --canary-input: var(--canary-base-chrome-150);
+    --canary-input-background: var(--canary-base-chrome-white);
+    --canary-ring: var(--canary-accent-400);
+
+    --canary-radius: 0.5rem;
+    --canary-success: var(--canary-yellow-600);
+    --canary-error: var(--canary-indigo-500);
+    --canary-warning: var(--canary-amber-700);
+    --canary-emphasis: var(--canary-accent-600);
+    --canary-ai: var(--canary-purple-700);
+    --canary-ai-button-stop-1: var(--canary-yellow-100);
+    --canary-ai-button-stop-2: var(--canary-pink-200);
+    --canary-ai-button-stop-3: var(--canary-purple-400);
+    --canary-ai-button-stop-4: var(--canary-cyan-200);
+
+    --diff-grey: hsl(var(--canary-base-chrome-50));
+    --diff-add-content--: hsl(var(--canary-yellow-50));
+    --diff-del-content--: hsl(var(--canary-indigo-50));
+    --diff-add-lineNumber--: hsl(var(--canary-yellow-100));
+    --diff-del-lineNumber--: hsl(var(--canary-indigo-100));
+    --diff-plain-content--: hsl(var(--canary-background-01));
+    --diff-expand-content--: hsl(var(--canary-background-01));
+    --diff-plain-lineNumber--: hsl(var(--canary-background-01));
+    --diff-plain-lineNumber-color--: hsl(var(--canary-base-chrome-850));
+    --diff-hunk-content--: hsl(var(--canary-accent-50));
+    --diff-hunk-lineNumber--: hsl(var(--canary-accent-100) / 0.7);
+    --diff-add-content-highlight--: hsl(var(--canary-yellow-100));
+    --diff-del-content-highlight--: hsl(var(--canary-indigo-100));
+    --diff-add-widget--: hsl(var(--canary-base-chrome-100));
+    --diff-add-widget-color--: hsl(var(--canary-base-chrome-850));
+    --diff-empty-content--: hsl(var(--canary-background-01));
+    --diff-hunk-content-color--: hsl(var(--canary-base-chrome-850));
+
+    /* New colors design variables */
+    /* Text */
+    --canary-foreground-01: var(--canary-base-chrome-black);
+    --canary-foreground-02: var(--canary-base-chrome-850);
+    --canary-foreground-03: var(--canary-base-chrome-600);
+    --canary-foreground-04: var(--canary-base-chrome-600);
+    --canary-foreground-05: var(--canary-base-chrome-600);
+    --canary-foreground-06: var(--canary-base-chrome-white);
+    --canary-foreground-07: var(--canary-base-chrome-600);
+    --canary-foreground-08: var(--canary-base-chrome-850);
+    --canary-foreground-09: var(--canary-base-chrome-600);
+    --canary-foreground-10: var(--canary-base-chrome-200);
+    --canary-foreground-danger: var(--canary-indigo-600);
+    --canary-foreground-alert: var(--canary-amber-700);
+    --canary-foreground-success: var(--canary-yellow-700);    
```

Looks like trailing spaces are contributing to the prettier failures.
```suggestion
    --canary-foreground-success: var(--canary-yellow-700);
```